wandb/sdk/wandb_setup.py

```diff
@@ -295,8 +295,9 @@ def _setup(settings=None, _reset=None):
     return wl
 
 
-def setup(settings=None):
+def setup(settings=None) -> Optional["_WandbSetup"]:
```

Comment:
same do you want to type this?

Reply:
This file is still not type-annotated :( I would leave it for a future PR, we should organize a type-annotation sprint or something.

Reply:
yeah that's why i nitted it haha